src/js/redirectCodeFragment.js

```diff
@@ -0,0 +1,8 @@
+export function onRouteUpdate({ location, previousLocation }) {
+  if (location.hash && location.pathname.includes("/reference/")) {
+    const matches = /^#code(.+)code$/.exec(location.hash);
```

Comment:
Can we add a comment explaining why we're doing this/where the `code...code` comes from?